src/db_util.py

```diff
@@ -99,3 +115,44 @@ def load_pynder_session(username):
 def create_pynder_session(fb_token, fb_id=None):
     pynder_session = pynder.Session(facebook_id=fb_id, facebook_token=fb_token)
     return pynder_session
+
+
+def add_vote(vote):
```

Comment:
fixed
modified add_vote and add_match

add_tinder_user has a precondition so it will be modified on a future refactoring